cmp/grid/ag-grid/ColumnHeader.js

```diff
@@ -37,6 +38,7 @@ export class ColumnHeader extends Component {
     // Get any active sortBy for this column, or null
     @computed
     get activeGridSorter() {
+        if (!this.gridModel) return;
```

Comment:
Need this check for the auto group col, which will get this header component but wont have a gridModel. Will explore better support in general for the auto group col as part of the pivot mode support.

Reply:
super abbreviated form of this comment in code?